db/migrate/20180123050321_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :users do |t|
+      t.string :username
+      t.string :email
```

Comment:
I think username and/or email should have
``null: false ``
to be sure all the users will enter one. (Don't they need a password as well? 🤔 )